src/main/models/exercise.ts

```diff
@@ -0,0 +1,88 @@
+'use strict';
+import { Model } from 'sequelize';
+
+export interface ExerciseAttributes {
+  id: number;
+  name: string;
+  description: string;
+  thumbnailSrc: string;
+  videoSrc: string;
+  length: number;
+}
+
+module.exports = (sequelize: any, DataTypes: any) => {
+  class Exercise
+    extends Model<ExerciseAttributes>
+    implements ExerciseAttributes
+  {
+    id!: number;
+    name!: string;
+    description!: string;
+    thumbnailSrc!: string;
+    videoSrc!: string;
+    length!: number;
+
+    static associate(models: any) {
+      Exercise.belongsToMany(models.MuscleGroup, {
+        through: 'MuscleGroupExercise',
+        onDelete: 'CASCADE',
```

Comment:
I'm not very familiar with squelize, could you clarify if onDelete cascade here means: if a MuscleGroup is deleted, then the Exercises in that MuscleGroup will also be deleted? If so, we might wanna onDelete set null?

Reply:
Yup! The `onDelete` here specifies that when an exercise is deleted from the `Exercise` table, all entries in `MuscleGroupExercise` associated with that particular exercise will be deleted as well- this doesn't impact the `MuscleGroup` table. This is technically already CASCADE by default because its n:m but I specified it just to be more clear! 

These links might help:
[Associations](https://sequelize.org/master/manual/assocs.html)
[Model](https://sequelize.org/v5/class/lib/model.js~Model.html)

Reply:
Thanks for clarifying, that makes sense!